client/app/components/History.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+const History = (props) => {
+  return (
+    <div>
+      <h2>History goes here</h2>
+      <ul>
+        <li>corgis</li>
```

Comment:
I assume that for the final version we'll be turning the li's into their own components and rendering them via a map.  Just want to confirm!